deploy.test.ts

```diff
@@ -1,5 +1,9 @@
+import { describe, expect } from "@jest/globals";
 import { deploy } from "./deploy";
 
+//jest.setTimeout(5000);
+jest.useFakeTimers();
+
 describe("Mock Deploy", () => {
   it("does not fail", async () => {
     await expect(() => {
```

Comment:
I was looking at this closer as the test still passes when I am not running a local frequency.

I think we need the test like this:

```suggestion
    await expect(deploy()).resolves.not.toThrow();
```

That then correctly errors out.

It however might require a few changes to fix. Thoughts on removing the test instead?

Reply:
Removing the test.